management/README.md

```diff
@@ -0,0 +1,14 @@
+# Azure Service Management packages for Go
+
+The *github.com/Azure/azure-sdk-for-go/management* packages are used to perform operations using the Azure Service Management (ASM), aka classic deployment model. Read more about [Azure Resource Manager vs. classic deployment](https://azure.microsoft.com/documentation/articles/resource-manager-deployment-model/). Packages for Azure Resource Manager are in the [arm](https://github.com/Azure/azure-sdk-for-go/tree/master/arm) folder.
```

Comment:
italics unnecessary make code blocks

also please use relative link for ./arm in markdown